firstvoices/backend/serializers/base_serializers.py

```diff
@@ -84,3 +85,16 @@ class CreateSiteContentSerializerMixin(CreateSerializerMixin):
     def create(self, validated_data):
         validated_data["site"] = get_site_from_context(self)
         return super().create(validated_data)
+
+
+class CreateSiteContentVisibilitySerializerMixin(CreateSiteContentSerializerMixin):
```

Comment:
Maybe CreateControlledSiteContentSerializerMixin, to match how the base models are named?